front-end/src/components/CenterConsole/SingleRecipe.js

```diff
@@ -8,7 +8,7 @@ const SingleRecipe = ({oneRecipe}) => {
 
   return (
     <Grid item xs={4}>
-      <Link to='/login'>
+      <Link to='/signup'>
```

Comment:
is the '/signup' link just a placeholder? or will it eventually route you to a recipe page?

Reply:
Hey @rick-o-H , ya , the current route is just a placeholder for now. It will eventually route to the recipe page like you mention and should be no problem to swap out once we iron out our routes.